dali/test/python/reader/test_webdataset_requirements.py

```diff
@@ -274,6 +274,150 @@ def test_pax_format():
         )
 
 
+def test_case_sensitive_container_format():
+    global test_batch_size
+    num_samples = 1000
+    tar_file_path = os.path.join(get_dali_extra_path(), "db/webdataset/MNIST/devel-0.tar")
+    case_insensitive_tar_file_path = os.path.join(get_dali_extra_path(),
+                                                  "db/webdataset/case_insensitive/devel-0.tar")
+    index_file = generate_temp_index_file(tar_file_path)
+
+    num_shards = 100
+    with assert_raises(AssertionError):
```

Comment:
nitpick: Can you add the glob/regex for the expected error message?

Reply:
Done